voting_rights/voting_rights_2016.csv

```diff
@@ -76,12 +76,12 @@ TN,2016,1,Strict photo ID,"At polling places in Tennessee, voters must present g
+ID; any federal, state, or local government-issued ID; employer-issued photo ID card; concealed handgun permit; student ID from any higher education institution in Virginia; or a current utility bill, bank statement, paycheck, or government check indicating the name and address of the voter. The identification must be current or expired only within the last year.",If voters do not have the proper ID they can cast a provisional ballot.,"Sept 22, 2016, A federal judge in May upheld Virginia's law, finding the state had “provided all of its citizens with an equal opportunity to participate in the electoral process” and that the voting rules are constitutional and do not violate the federal Voting Rights Act. All three judges were nominated by Republican presidents. News from 12/24/16 The Virginia law allows people who show up without a government-issued or otherwise-approved photo ID to vote, but they have to return within three days with their ID for it to count. Voters without an ID can obtain one for FREE from board of elections offices around the state."
 WA,2016,1,Non-strict non-photo ID,"Washington is a vote-by-mail state. County auditors are required to open at least one vote center where voters can cast a ballot in person, and a valid photo ID is required.",No if voting in person but most people vote by mail.,No pending legislation.
 WV,2016,0,Non-strict non-photo ID; goes into effect in 2018,First-time voters in West Virginia who registered by mail must present identification. Valid identification includes photo and non-photo identification.,Yes unless you are a first-time voter.,There was legislative movement in 2016. Nothing new appears to be in the news.
 WI,2016,1,Strict photo ID ,Voters in Wisconsin are required to present photo identification at the polls.,No.,"July 2016 in Wisconsin, a federal judge ruled that voters without ID must still be allowed to vote, saying that a state law, ostensibly intended to prevent voter fraud, led instead to disenfranchisement. But a footnote for all ' he three states that sunk Clinton: Pennsylvania, Michigan, and 
```

Comment:
@augie can you re-run your script with the csv from git? we want to make sure the csv is still working with your script..thank you!